dev/com.ibm.ws.java11_fat/publish/servers/server_MultiReleaseJarTest/server.xml

```diff
@@ -0,0 +1,22 @@
+<server>
+
+    <featureManager>
+        <feature>componenttest-1.0</feature>
+        <feature>servlet-4.0</feature>
+    </featureManager>
+    
+    <include location="../fatTestPorts.xml"/>
+    
+    <application location="multiReleaseApp.war" />
+    
+    <application location="multiReleaseSharedLibApp.war">
+        <classloader commonLibraryRef="MRJarLib"/>
+    </application>
+    
+    <library id="MRJarLib">
+        <fileset dir="${server.config.dir}/lib"/>
```

Comment:
I assume the jar is built and committed to git because we don't have a mechanism today to build MR jars in simplicity yet?  Are the classes in it built with the right byte code level in order to support on multiple jdk levels? I assume you have tested on Java 8 and above with what is committed to git.

Reply:
yea, putting class files into `META-INF/versions/X/` was not easily doable with ShrinkWrap, so I just checked in the (~100KB) jar to git. The test classes in the MR jar are so trivial I doubt they will ever need to change.

I tested this locally with Java 8, 11, and 12.